maps/map05.txt

```diff
@@ -1,7 +1,12 @@
-##########
-#o.......#
-#.       #
-#.       #
-#.       #
-#.      g#
-##########
+#### ####
+#.  .  .#
+# ## ## #
+# ..... #
+# ## ## #
+# ..... #
+# ## ## #
+    o    
+# ## ## #
+#. . . .#
+#g  .   #
+#### ####
```

Comment:
I've added a little bit more pills there. Especially those at the bottom are hard.